### PR TITLE
fix(dropdown): prevent activating disabled dropdown item (and minor cleanup)

### PR DESCRIPTION
**Pull request by kevinbuhmann**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
  - I couldn't get enter/space key tests to work.
- [N/A] Docs have been added / updated (for bug fixes / features)
- [N/A] If applicable, have a visual design approval

## PR Type

Bugfix

## What is the current behavior?

Disabled dropdown items can be activated via click, space, and enter.

Issue Number: #387 and #389

## What is the new behavior?

Disabled dropdown items cannot be activated via click, space, and enter.

## Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: projects/angular/src/popover/dropdown/dropdown-item.ts [hunk 1]
```diff
@@ -70,19 +67,10 @@ export class ClrDropdownItem implements AfterViewInit {
     return this.focusableItem.id;
   }
 
-  ngAfterViewInit() {
-    this.unlisten = this.renderer.listen(this.el.nativeElement, 'click', () => this.onDropdownItemClick());
```
Comment (Jinnie) [inline projects/angular/src/popover/dropdown/dropdown-item.ts:74]: I try to remember or find a reason to have it done this way, but I can't.

**Review by Jinnie — APPROVED** [review]

File: projects/angular/src/popover/dropdown/dropdown-item.ts (hunk 1 above)
Comment (Jinnie) [reply projects/angular/src/popover/dropdown/dropdown-item.ts:74]: Oops, there is a failing test - it may be a clue.

**Review by Jinnie — CHANGES_REQUESTED** [review]
Please check the failing test scenario.

File: projects/angular/src/popover/dropdown/dropdown-item.ts (hunk 1 above)
Comment (kevinbuhmann) [reply projects/angular/src/popover/dropdown/dropdown-item.ts:74]: Yeah, I think handler is added in `ngAfterViewInit` to make sure it run last. I am looking into it.

Comment (kevinbuhmann) [reply projects/angular/src/popover/dropdown/dropdown-item.ts]: I dropped this commit. I might open a separate PR.

Comment (kevinbuhmann) [reply projects/angular/src/popover/dropdown/dropdown-item.ts:74]: I fixed it with `setTimeout`.

**Review by Jinnie — APPROVED** [review]
Thanks

**Pull request merged**